### PR TITLE
correction for flutter analyze in reading_writing_files.md

### PR DESCRIPTION
**Pull request by hrishikesh-kadam**
Related to #5786

### CODE REVIEW

**Comment by miquelbeltran** [comment]
LGTM

File: null_safety_examples/cookbook/persistence/reading_writing_files/test/widget_test.dart [hunk 1]
```diff
@@ -24,3 +32,19 @@ void main() {
     // #enddocregion setUpAll
   });
 }
```
Comment (hrishikesh-kadam) [inline null_safety_examples/cookbook/persistence/reading_writing_files/test/widget_test.dart:34]: `flutter test` for just this existing test was resulting in - `No tests ran.`, which was failing the `tool/check-code.sh --null-safety`

**Comment by hrishikesh-kadam** [comment]
`flutter test` for existing [L15-L33][] test was resulting in - `No tests ran.`, which was failing the `tool/check-code.sh --null-safety`.

So added [L10-L13][] test from [path_provider/example/integration_test/path_provider_test.dart][], which results in - `00:02 +1: All tests passed!` and doesn't fail the `tool/check-code.sh --null-safety`.

[L15-L33]: https://github.com/flutter/website/blob/6ac26055948e94a86ac25b907fc6f5398d782eb1/null_safety_examples/cookbook/persistence/reading_writing_files/test/widget_test.dart#L15-L33
[L10-L13]: https://github.com/flutter/website/blob/6ac26055948e94a86ac25b907fc6f5398d782eb1/null_safety_examples/cookbook/persistence/reading_writing_files/test/widget_test.dart#L10-L13
[path_provider/example/integration_test/path_provider_test.dart]: https://github.com/flutter/plugins/blob/015a77290c85d0fc2fcc0c96d7f6c1fb67353fe3/packages/path_provider/path_provider/example/integration_test/path_provider_test.dart

**Comment by hrishikesh-kadam** [comment]
@sfshaza2 Can you please merge this too?

**Comment by miquelbeltran** [comment]
Makes sense, thanks! the previous code was just including the code for the sake of extracting it into the markdown, but since it was not even running any tests it was weird.

**Pull request merged**